src/main.rs

```diff
@@ -621,7 +635,8 @@ fn main() {
                 args.annotate_attribute_only,
                 args.whole_document,
                 args.whole_paragraphs,
-                &pbar
+                &pbar,
+                args.no_progress,
```

Comment:
Can `Arc` pointers be null? Then you could save a parameter and pass in null here instead.

Reply:
Yes...? Pretty sure we can wrap with an option and then we don't have to pass the second parameter